src/EditorFeatures/CSharp/Wrapping/BinaryExpression/CSharpBinaryExpressionWrapper.cs

```diff
@@ -13,6 +14,9 @@ public CSharpBinaryExpressionWrapper()
         {
         }
 
+        protected override IBlankLineIndentationService GetIndentationService()
+            => new CSharpIndentationService();
```

Comment:
consider exposing and using a singleton here.  THat's generally what we've done with other services.  i.e. CSharpSyntaxFactsService.Instance.

Reply:
This pattern is carried over from before. I'll file an issue to address separately if it's not fixed before merge.

Reply:
➡️ Moved to #32831